pkg/asset/internal/templates.go

```diff
@@ -1174,24 +1174,34 @@ data:
   # The CNI network configuration to install on each node.
   cni_network_config: |-
     {
-        "name": "k8s-pod-network",
-        "cniVersion": "0.3.0",
-        "type": "calico",
-        "log_level": "debug",
-        "datastore_type": "kubernetes",
-        "nodename": "__KUBERNETES_NODE_NAME__",
-        "ipam": {
+      "name": "k8s-pod-network",
+      "cniVersion": "0.3.0",
```

Comment:
This should be 0.3.1 - we fixed a small bug.

Reply:
Sure thing